mono/metadata/class.c

```diff
@@ -5986,7 +5996,7 @@ mono_class_is_nullable (MonoClass *klass)
 }
 
 
-/** if klass is T? return T */
+/** For a particular Nullable<T>, get the class of the T */
```

Comment:
Honestly the original wording confused me because I wasn't sure if T? was a unicode error ^_^; but I can change it back.